src/BLSSignatureChecker.sol

```diff
@@ -73,6 +73,10 @@ contract BLSSignatureChecker is IBLSSignatureChecker {
         // loop through every quorumNumber and keep track of the apk
         BN254.G1Point memory apk = BN254.G1Point(0, 0);
         for (uint i = 0; i < quorumNumbers.length; i++) {
+            require(
+                registryCoordinator.quorumUpdateTimestamp(uint8(quorumNumbers[i])) + 1 weeks <= block.timestamp,
```

Comment:
should we set a minimum withdrawalDelayBlocks in delegation manager and set this to that?

cc @ChaoticWalrus

Reply:
what if we just read the current `DelegationManager.withdrawalDelayBlocks` value instead?

Reply:
oh yup! that'l work

Reply:
Added this in the feature PR https://github.com/Layr-Labs/eigenlayer-middleware/pull/62/commits/e4f39f61b962319578e8d8e4c0ef4582084833c7 as well as the interface change to read off of the DM https://github.com/Layr-Labs/eigenlayer-contracts/pull/343